pkg/policy/validate_controller.go

```diff
@@ -180,6 +180,9 @@ func (pc *PolicyController) canBackgroundProcess(p *kyverno.ClusterPolicy) bool
 func (pc *PolicyController) addPolicy(obj interface{}) {
 	logger := pc.log
 	p := obj.(*kyverno.ClusterPolicy)
+
+	logger.Info("policy created event", "uid", p.UID, "kind", "Policy", "policy_name", p.Name)
```

Comment:
I should make it clearer in the last comment. In Kyverno, we have ClusterPolicy and Policy, similar to ClusterRole and Role, the ClusterPolicy is a cluster-wide resource and the Policy is a namespaced source.

So here the kind should be ClustserPolicy, or you can also use `p.Kind` to get the kind.

Reply:
When I first started, I used `p.Kind` to get kind, but I found that what I got was empty, so I changed it to a fixed value. I understand what you mean, I will modify it again according to the correct kind type.